core/src/main/java/org/geysermc/geyser/session/cache/InputCache.java

```diff
@@ -122,7 +147,14 @@ public boolean lastHorizontalCollision() {
         return lastHorizontalCollision;
     }
 
-    public void setLastHorizontalCollision(boolean lastHorizontalCollision) {
-        this.lastHorizontalCollision = lastHorizontalCollision;
```

Comment:
I think this was removed by mistake as its still used in `BedrockMovePlayer`
If its no longer needed then I suggest we cleanup the other function for getting it along with the variable.

Reply:
The field is now marked with a Lombok setter instead